### PR TITLE
[API] Add missing Admin Orders API contract tests

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13                   |
| Bug fix?        | no                                                      |
| New feature?    | no                                                      |
| BC breaks?      | no                                                      |
| Deprecations?   | no |
| License         | MIT                                                          |

We aim to achieve maximum API test coverage before the Sylius 1.13 release.

This PR enhances testing by introducing some utility functions

### CODE REVIEW

File: tests/Api/Admin/OrdersTest.php [hunk 1]
```diff
@@ -79,17 +114,109 @@ public function it_gets_orders_for_customer(): void
...
+    /**
+     * @test
+     *
+     * @dataProvider provideDates
+     *
+     * @param string[] $checkoutsCompletedAt
+     * @param array{filterType: FilterTypes, date: string} $requestedLimit
+     *
+     * @throws \Exception
+     */
```
Comment (Wojdylak) [inline tests/Api/Admin/OrdersTest.php]: This is a test. Are these comments necessary?

File: tests/Api/Admin/OrdersTest.php [hunk 2]
```diff
@@ -79,17 +114,109 @@ public function it_gets_orders_for_customer(): void
...
+    public function it_gets_orders_by_period(
+        string $tokenValue,
+        array $checkoutsCompletedAt,
+        array $requestedLimit,
+        ?string $filename = null,
```
Comment (Wojdylak) [inline tests/Api/Admin/OrdersTest.php]: Why is this nullable?

File: tests/Api/Admin/OrdersTest.php [hunk 3]
```diff
@@ -79,17 +114,109 @@ public function it_gets_orders_for_customer(): void
...
+
+        $this->requestGet(
+            uri: '/api/v2/admin/orders',
+            queryParameters: [$checkoutCompletedAt => $requestedLimit['date']],
         );
+
+        $this->assertResponseSuccessful($filename);
+    }
+
+    private function provideDates(): iterable
```
Comment (Wojdylak) [inline tests/Api/Admin/OrdersTest.php]:
```suggestion
    private function provideOrderFilterDates(): iterable
```

File: tests/Api/Admin/OrdersTest.php (hunk 1 above)
Comment (Rafikooo) [reply tests/Api/Admin/OrdersTest.php]: IDE complains, but perhaps I should configure it to exclude test namespaces.

File: tests/Api/Admin/OrdersTest.php (hunk 2 above)
Comment (Rafikooo) [reply tests/Api/Admin/OrdersTest.php]: Mistake

**Review by NoResponseMate — APPROVED** [review]

**Comment by NoResponseMate** [comment]
Thank you, @Rafikooo!

**Pull request merged**